openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -453,16 +463,19 @@ function processIndicators(layerAttributes, projectDef) {
                 la[ix].newProperties['SVI'] = (SVI[key]).toFixed(5);
             }
         }
-        for (var key in RI) {
-            if (key == la[ix].properties[selectedRegion]) {
-                la[ix].newProperties['IR'] = (RI[key]).toFixed(5);
+
+        if (riskIndicators !== undefined) {
```

Comment:
Let's always check undefined stuff like `if (typeof riskIndicators !== 'undefined') {`
(please search and modify all possible occurrences of that, accordingly)